feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Hours.kt

```diff
@@ -269,11 +269,13 @@ private class HoursScreen(
         position: Offset,
     ) {
         val nextPossibleY = calculatePossibleScrollY(dragAmount.y)
-        scrollState.scroll(
-            Offset(scrollState.scrollX, nextPossibleY),
-            timeMillis,
-            position
-        )
+        if (scrollState.scrollX.isNaN().not() && nextPossibleY.isNaN().not()) {
```

Comment:
Looks good! If we have time we can create `ScrollState.safeScroll()` extension function.

Reply:
I couldn't react to your comment immediately, and I couldn't reflect this comment.
Thank you for your proposal.